script/bootstrap

```diff
@@ -7,7 +7,22 @@ set -e
 
 cd "$(dirname "$0")/.."
 
+echo "==> Installing Ruby…"
+rbenv install --skip-existing
+
 echo "==> Installing gem dependencies…"
+which bundle 2>&1 > /dev/null || {
+  gem install bundler
+}
 bundle check --path vendor/gems 2>&1 > /dev/null || {
   bundle install --path vendor/gems --quiet --without production
 }
+
+if [ "$(uname -s)" = "Darwin" ] && brew --prefix 2>&1 > /dev/null; then
+  echo "==> Installing Homebrew dependencies…"
+  brew update
+  brew tap homebrew/bundle 2>/dev/null
+  brew bundle check 2>&1 > /dev/null || {
```

Comment:
So, my problem with that is that this file/repository assumes a Gemfile exists in the root directory even although one is included in the repo. If you didn't have a `Gemfile` because this was e.g. a NodeJS project then you wouldn't add a `Gemfile` check, you'd just not include this code. This feels like the same thing to me with the `Brewfile`: if you weren't using one for this repository you'd just not include this code rather than guard it to be a no-op if it doesn't exist.

I think this differs from the OS check because you may well have a `Brewfile` present for Linux machines but don't want it run (and there's no obvious Linux alternative yet as far as I can tell).